orchestration/templates/cron-workflow.yaml

```diff
@@ -15,12 +15,14 @@ spec:
     parallelism: {{ .Values.maxParallelism }}
     templates:
       - name: main
-        templateRef:
-          name: ingest-clinvar-archive
-          template: main
-        inputs:
-          parameters:
-            - name: archive-path
-              value: weekly_release/ClinVarVariationRelease_00-latest_weekly.xml.gz
-            - name: gcs-prefix
-              value: '{{ template "argo.timestamp" }}'
+        steps:
```

Comment:
Argo has deprecated top-level `templateRef` usage within tasks. All refs must be migrated under either `steps` or a `dag`